frontend/openapi-ts.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from '@hey-api/openapi-ts';
+
+export default defineConfig({
+  input: 'http://localhost:8888/openapi.json',
+  output: './src/lib/api-client',
+  plugins: ['legacy/fetch'],
+  name: 'ApiClient',
```

Comment:
The legacy client is deprecated now, but upgrading these to `@hey-api/client-fetch` looks like it's a big breaking change.